rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,170 @@
+
+    ```{bash}
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+    sudo add-apt-repository 'deb https://cloud.r-project.org/bin/Linux/ubuntu focal-cran40/'
+    ```
+
+    * **Note:** More detailed information on installing the latest R for Linux can be found at [the official CRAN page](https://cran.r-project.org/bin/Linux/ubuntu/){target="_blank"}.
+
+1. Install R with some essential dependencies for the RStudio Server and devtools:
+
+    ```{bash}
+    sudo apt install -y r-base r-base-core r-recommended r-base-dev gdebi-core build-essential libcurl4-gnutls-dev libxml2-dev libssl-dev
+    ```
+
+1. Install RStudio Server:
+
+    ```{bash}
+    wget https://rstudio.org/download/latest/stable/server/bionic/rstudio-server-latest-amd64.deb
+    sudo gdebi rstudio-server-latest-amd64.deb
+    ```
+
+    * **Note1:** You may receive a warning message like "Couldn't find an alternative telinit implementation to spawn." However, our testing results have revealed that there is no issue with using RStudio Server regardless of this warning.
+
+    * **Note2:** While this article has demonstrated the installation of the current stable release, you can also try with other versions by consulting with some RStudio download pages, such as [RStudio Release](https://rstudio.com/products/rstudio/download/#download){target="_blank"}, [RStudio Preview](https://rstudio.com/products/rstudio/download/preview/){target="_blank"}, and [RStudio Daily](https://dailies.rstudio.com/){target="_blank"}. Make sure you choose RStudio *Server* (not Desktop) and match with your installed Linux distribution type and version.
+
+1. Launch RStudio Server:
+
+    ```{bash}
+    sudo rstudio-server start
+    ```
+
+    * **Note1:** If there was no error message, you can assume that your RStudio Server has been successfully loaded.
+
+    * **Note2:** If you terminate and restart WSL, you will need to run `sudo rstudio-server start` again to get RStudio Server up and running.
+      
+1. Access your server at [http://localhost:8787](http://localhost:8787){target="_blank"}.
+
+    * **Note:** The user name and password are the same as your Linux  system's.
+
+## Removing RStudio Server {#remove-rstudio-server}
+
+Sometimes you may want to remove the installed RStudio Server. For example, you have to do this in order to upgrade your RStudio Server.
+
+The following steps explain how to safely achieve this goal:
+
+1. Press `Windows` logo key to open start menu, then type `wsl` to launch your installed Linux distro.
+1. To stop RStudio Server, type the following command in the WSL terminal:
+
+    ```{bash}
+    sudo rstudio-server stop
+    ```
+
+    * **Note:** Provide the password of your user account that you created for the Linux system, not the Windows password to log in for your PC.
+
+1. Remove your Server:
+
+    ```{bash}
+    sudo apt-get remove rstudio-server -y
+    ```
+
+
```

Comment:
Maybe add another section titled:

## Upgrading RStudio Server

You could link to this article that discusses how to upgrade RStudio Server, though I worry it has way too many details that WSL users won't need to care about. Up to you whether you think this section is worth adding at all!

https://support.rstudio.com/hc/en-us/articles/216079967-Upgrading-RStudio-Server